kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -208,6 +205,27 @@ extension [A, S, E](effect: A < (Abort[E] & S))
             .map(_.fold(e => throw e.getFailure)(identity))
 end extension
 
+extension [A, S, E](effect: A < (Abort[Maybe.Empty] & S))
+    def handleEmptyAbort(using f: Flat[A]): Maybe[A] < S =
```

Comment:
I've tried to be consistent with the method names for handling and converting error types. Also, this method doesn't really "unmaybe" anything, does it? It produces a `Maybe`.

Reply:
I was thinking that combinators would follow ZIO's general naming conventions, eg https://github.com/zio/zio/blob/series/2.x/core/shared/src/main/scala/zio/ZIO.scala#L2316-L2322

I guess this would be `unmaybe`